src/screens/SurveyItem/Comment/index.tsx

```diff
@@ -74,7 +73,6 @@ const Comment: React.FC<Props> = ({surveyId, commentItem, refetch}) => {
         DislikeCommentMutationVariables
     >(DISLIKE_COMMENT, {
         onCompleted: () => {
-            Toast.show('Comment disliked Successfully !');
```

Comment:
What will happen to replied comments if the parent one is deleted?

Reply:
currently, the delete feature is not implemented yet. On the server side, if the parent comment object is deleted, also delete the objects that have references to it

Reply:
Maybe we can add an is_deleted attribute to the and just flag those deleted comments. On the mobile side, need to use proper design so user understands that the comment is deleted.